helpdesk-service/helpdesk-service-server/src/test/java/org/symphonyoss/symphony/bots/helpdesk/service/steps/MakerCheckerSteps.java

```diff
@@ -0,0 +1,243 @@
+package org.symphonyoss.symphony.bots.helpdesk.service.steps;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertNull;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+
+import org.jbehave.core.annotations.Given;
+import org.jbehave.core.annotations.Then;
+import org.jbehave.core.annotations.When;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.symphony.bots.helpdesk.service.makerchecker.client.MakercheckerClient;
+import org.symphonyoss.symphony.bots.helpdesk.service.model.Makerchecker;
+import org.symphonyoss.symphony.bots.helpdesk.service.model.UserInfo;
+
+/**
+ * Created by alexandre-silva-daitan on 23/02/18.
+ */
+
+@Component
```

Comment:
Why this annotation?

Reply:
As a component, I can use the Autowired in the class for the restTemplate